etc/config/javascript.amazon.properties

```diff
@@ -0,0 +1,16 @@
+compilers=javascript
+defaultCompiler=javascript
+
+supportsBinary=false
+interpreted=true
+compilerType=v8
+objdumper=
+demangler=
+postProcess=
+options=
+supportsExecute=false
+stubText=function add(a, b) { return a + b; } for (var i = 0; i < 10; i++) { add(i, 1) }
+
+compiler.javascript.exe=/opt/compiler-explorer/d8-11.1/out/x64.release/d8
```

Comment:
We are also missing a PR to infra to install a version of d8: I realised our docs are super out of date (oops) so I'll see if I can muddle something together.

Reply:
Thankyou